src/main/webapp/sass/_core.scss

```diff
@@ -134,7 +134,12 @@ a:focus{color:$synapse-blue;}
 }
 
 .SynapseNavDrawerIsShowing {
-	padding-left: 53px;
+    $-nav-drawer-width: 53px;
+	padding-left: $-nav-drawer-width;
+
+	.SynapseToastContainer {
+	    width: calc(100% - #{$-nav-drawer-width});
+	}
```

Comment:
This ensures the toast message doesn't cover the nav. Worked for me in FF/Chrome/Safari

Reply:
Nice!  That explains why it is the correct width.  What makes it properly horizontally aligned?

Reply:
The horizontal positioning is based on its location in the DOM, (it's at the same level as the headerPanel, rootPanel, and footerPanel). If width is set to 100%, it's still properly left-aligned, but the right side of the toasts will go past the width of everything else.